test/41-device-api-v1.spec.ts

```diff
@@ -728,121 +729,429 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 		});
 	});
 
-	describe('GET /v1/device/host-config', () => {
-		it('returns the current host config', async () => {
-			// TODO: test with proxy values (need to add redsocks conf path to test env)
-			await request
-				.get('/v1/device/host-config')
-				.set('Accept', 'application/json')
-				.set('Authorization', `Bearer ${apiKeys.cloudApiKey}`)
-				.expect(sampleResponses.V1.GET['/device/host-config'].statusCode)
-				.then((response) => {
-					expect(response.body).to.deep.equal(
-						sampleResponses.V1.GET['/device/host-config'].body,
-					);
-				});
-		});
-	});
-
-	describe('PATCH /v1/device/host-config', () => {
-		// TODO: add tests for proxy PATCH reqs - probably need to add test files for redsocks.conf or other
-
+	describe('/v1/device/host-config', () => {
```

Comment:
Same here, is probably better to squash this commit with 18f03a0

Reply:
Done!